sdks/java/io/azure/build.gradle

```diff
@@ -17,7 +17,11 @@
  */
 
 plugins { id 'org.apache.beam.module' }
-applyJavaNature(automaticModuleName: 'org.apache.beam.sdk.io.azure')
+applyJavaNature(automaticModuleName: 'org.apache.beam.sdk.io.azure', enableChecker: false)
 
 dependencies {
+  compile "com.azure:azure-storage-blob:12.6.0"
+  compile project(path: ":sdks:java:core", configuration: "shadow")
+  compile "com.azure:azure-identity:1.0.8"
```

Comment:
Let's remove this dependency for now, as we're not really using it.
Let's also remove azure-storage-blob:12.6.0 until we start using it.

Reply:
Which dependency?  Just the 1st and 3rd, or also the 2nd?

Reply:
Any dependency that we're not using. I believe that's `compile "com.azure:azure-storage-blob:12.6.0"` and `compile "com.azure:azure-identity:1.0.8"` - 

`:sdks:java:core` contains the base classes for filesystem, so we keep that one, and junit.

Reply:
Got it, thanks.